### ISSUE TITLE
[Bug] GPT-OSS raw converter emits incorrect expert weight format, causing gibberish output with `--megatron-to-hf-mode raw`

### ISSUE THREAD

**Issue by qqqlm** [THUDM/slime#1840]
### Bug Description

![convert_gpt_oss_to_hf](https://github.com/THUDM/slime/blob/8efb1166c8e4bb4c810059b0161cbd06bcfbb6cf/slime/backends/megatron_utils/megatron_to_hf/gpt_oss.py#L6) has two bugs when using `--megatron-to-hf-mode raw` (non-colocate). After the first weight sync, SGLang produces gibberish (reward=0, 100% truncation).

1. **fc1**: Emits separate `gate_proj`/`up_proj` per expert instead of fused interleaved `gate_up_proj`. SGLang expects fused 3D tensors via ![`make_expert_params_mapping_fused()`](https://github.com/sgl-project/sglang/blob/bbe9c7eeb520b0a67e92d133dfc137a3688dc7f2/python/sglang/srt/models/gpt_oss.py#L1022).
2. **fc2**: Missing transpose — Megatron `(out, in)` vs HF `(in, out)`. Hidden in 20B where `hidden == inter == 2880`.

![`scripts/run-gpt-oss-20B.sh`](https://github.com/THUDM/slime/blob/8efb1166c8e4bb4c810059b0161cbd06bcfbb6cf/scripts/run-gpt-oss-20B.sh#L151) uses `--megatron-to-hf-mode bridge --colocate`, bypassing ![convert_gpt_oss_to_hf](https://github.com/THUDM/slime/blob/8efb1166c8e4bb4c810059b0161cbd06bcfbb6cf/slime/backends/megatron_utils/megatron_to_hf/gpt_oss.py#L6) entirely, so this path was never tested.

### Steps to Reproduce

1. Use `origin/main` (verified at `8efb1166`)
2. Run GPT-OSS 20B GRPO with `--megatron-to-hf-mode raw` (non-colocate, 2-node: training TP=1/PP=2/EP=4, rollout SGLang TP=2/DP-attn=4)
3. Observe rollout after first weight sync

### Expected Behavior

Coherent responses, `truncated_ratio < 1%`, `reward > 0`

### Actual Behavior

```
response_len/mean: 2048.0   (all hit max length)
truncated_ratio:   1.0       (100%)
reward:            0
```

Broken code:
```python
# fc1 — separate unfused (wrong)
gate_weight, up_weight = param.chunk(2, dim=0)
return [
    (f"...experts.{expert_idx}.gate_proj.weight", gate_weight),
    (f"...experts.{expert_idx}.up_proj.weight", up_weight),
]
# fc2 — missing transpose
return [(f"...experts.{expert_idx}.down_proj.weight", param)]
```

Fix:
```python
# fc1 — fused interleaved
gate, up = param.chunk(2, dim=0)
fused = torch.stack([gate, up], dim=1).reshape(-1, param.shape[-1])
fused = fused.transpose(0, 1).contiguous()
return [(f"...experts.{expert_idx}.gate_up_proj.weight", fused)]
# fc2 — transposed
return [(f"...experts.{expert_idx}.down_proj.weight", param.T.contiguous())]
```

After fix: `response_len/mean: 427`, `truncated_ratio: 0.004`, `reward: 0.078`, `grad_norm: 0.377`.

The weight sync pipeline also needs to accumulate per-expert 2D tensors into fused 3D `[num_experts, ...]` before sending to SGLang, since `weight_loader_fused()` only accepts 3D shapes.

### Environment

slime version: `origin/main` at `8efb1166`
- Python version: 3.10
- PyTorch version: 2.6.0
- CUDA version: 12.8
- GPU type and count: 16x H200
- OS: Ubuntu 22.04 (Docker)
- SGLang version: v0.5.9 (from `slimerl/sglang:v0.5.9` base image)
- Megatron-LM version: commit `3714d81d`

### Logs

```shell

```

### Pre-submission Checklist

- [x] I have read the [CONTRIBUTING.md](https://github.com/THUDM/slime/blob/main/CONTRIBUTING.md) and understand the collaboration scope.
- [x] I have read the [documentation](https://thudm.github.io/slime/) and my issue is not addressed there.
- [x] I have searched for [existing issues](https://github.com/THUDM/slime/issues) and this is not a duplicate.
- [x] I have provided a minimal, reproducible example.
